tests/test_roxarapi/test_roxarapi_reek.py

```diff
@@ -125,6 +125,7 @@ def fixture_create_project(tmpdir, roxinstance) -> str:
     logger.info("Initialised RMS project, done!")
 
     poi = xtgeo.points_from_file(POINTSDATA2, fformat="rms_attr")
+    print(poi.dataframe.dtypes)
```

Comment:
You probably want to add some assert instead of a print statement here to check for the expected dtype.

Reply:
It is a test fixture so print was only debugging. Now deleted